src/core_landice/Registry.xml

```diff
@@ -165,6 +165,42 @@
                         description="If true, then ice from small grounded islands is removed.  Specifically, this finds one- and two-cell masses of ice that are surrounded by open ocean and eliminates them by sending them to the calving flux.  Grounded such ice masses would be islands, floating would be icebergs.  The ice removed is added to the calvingThickness field."
 	                possible_values=".true. or .false."
 		/>
+		<nml_option name="config_restore_damage" type="logical" default_value=".false." units="unitless"
+                        description="If true, then turn on the restore_damage function."
+		            possible_values=".true. or .false."
+                />
+		<nml_option name="config_calculate_damage" type="logical" default_value=".false." units="unitless"
+                        description="If true, then turn on the calculate_damage function."
+		            possible_values=".true. or .false."
+                />
+                <nml_option name="config_damage_restore_threshold" type="real" default_value="0.0" units="unitless"
+                        description="It is a value above which the old damage number would be kept from the impact of advection (the damage would not heal). For example, if the damage at some location is 0.83, and we set config_damage_restore_threshold to 0.8, then the damage number there will not fall below 0.83. Otherwise, if the damage is 0.7, then it could be decreased to 0.6 if non-damaged ice is transported from upstream. If we set config_damage_restore_threshold to 0 (default), then it will just not play a role in pratical."
+	                possible_values="real value between 0 and 1"
+		/>
+                <nml_option name="config_damage_calving_threshold" type="real" default_value="1.0" units="unitless"
+                        description="It is a value above which the cell would be calved away."
+	                possible_values="real value between 0 and 1"
+		/>
+                <nml_option name="config_damage_stiffness_min" type="real" default_value="0.1" units="unitless"
+                        description="When coupling damage to rheology, we do not allow it to fall below this value"
+	                possible_values="real value between 0 and 1"
+		/>
+                <nml_option name="config_damage_rheology_coupling" type="logical" default_value=".false." units="unitless"
+                        description="If true, then we change stiffnessFactor to 1-damage during the run."
+	                possible_values=".true. or .false."
+		/>
+                <nml_option name="config_damage_use_constant_A" type="logical" default_value=".false." units="unitless"
+                        description="If true, then we use constant A (i.e., config_default_flowParamA) in the damage calculation."
+	                possible_values=".true. or .false."
+		/>
+                <nml_option name="config_damage_if_calving_rate" type="logical" default_value=".false." units="unitless"
```

Comment:
I agree Matt that this would be a more intuitive / clear way to specify this option.

Reply:
> I'm thinking it might be less confusing to have an option called something like `config_damage_calving_method` that can be set to either `threshold` or `rate`.
> 
> Actually, it looks like this option is not used anywhere in your changes. Does it need to be removed? Are there still two ways to convert damage to calving, or just one?

Now I use config_damage_calving_method. The threshold method wasn't in the code and I've added it back and we can choose to use either the calving_rate method or the threshold method in the dama_calving subroutine.